.github/actions/install-dependencies/action.yml

```diff
@@ -18,8 +18,13 @@ runs:
         key: ${{ runner.os }}-blade-${{ hashFiles('**/yarn.lock') }}
 
     # cache miss - install packages with `yarn --frozen-lockfile`
-    # cache hit - download packages from the cache
+    # cache hit - download packages from the cache and explicitly run postinstall script
     - name: install packages if cache miss
       run: yarn --frozen-lockfile
       shell: bash
       if: steps.cache.outputs.cache-hit != 'true'
+
+    - name: Run postinstall script
```

Comment:
Why do we need to run this explicitly? won't yarn run it anyway? Or is it that when there is cache-hit, it will assume yarn is done and skip post install?

Reply:
We don't run `yarn` when it's cache-hit. Hence it's required.